exopy_hqc_legacy/tasks/tasks/instr/views/transfer_awg_file_task_view.enaml

```diff
@@ -0,0 +1,398 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by EcpyPulses Authors, see AUTHORS for more details.
```

Comment:
Copyright

Reply:
And same comment as for the task itself we can probably reuse some parts of the other tasks (in particular none of the functions looks like they need to be duplicated).